assignment.js

```diff
@@ -30,9 +30,11 @@ Use the following test cases to confirm your program meets the success criteria
 ************/
 console.log('Problem 1:')
 
-// Add your code below this line
-
-// Add your code above this line
+const baseNumber = 2
+for(let i = 0; i <= 12; i++){
+  newNumber = baseNumber * i
+  console.log(newNumber)
+}
 
```

Comment:
@nmikhol Good job using the for loop to complete the task. Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L38)